Data/src/main/java/org/tribuo/data/columnar/ResponseProcessor.java

```diff
@@ -47,13 +50,36 @@
      * @param fieldName The field name.
      *
      */
-    @Deprecated()
+    @Deprecated
     public void setFieldName(String fieldName);
 
     /**
+     * @deprecated use {@link #process(List)} and support multiple values instead.
      * Returns Optional.empty() if it failed to process out a response.
      * @param value The value to process.
      * @return The response value if found.
      */
+    @Deprecated
     public Optional<T> process(String value);
+
+    /**
+     * Returns Optional.empty() if it failed to process out a response. Though this method currently has
+     * a default implementation, it is the core method and should be overridden by implementors. Unless it
```

Comment:
I think I'd prefer "This method has a default implementation for backwards compatibility with Tribuo 4.0 and 4.1. This method should be overridden by code which depends on newer versions of Tribuo. The default implementation will be removed when the deprecated members are removed." to make it explicit.